src/main/java/Main.java

```diff
@@ -0,0 +1,53 @@
+import java.io.*;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Manager manager = new Manager();
+        try {
+            FileInputStream fileInputStream = new FileInputStream("src/main/resources/save.ser");
+            ObjectInputStream objectInputStream = new ObjectInputStream(fileInputStream);
+            manager = (Manager) objectInputStream.readObject();
+        } catch (IOException e) {
+            System.out.println("File save.ser does not exist");
+        } catch (ClassNotFoundException e) {
+            e.printStackTrace();
+        }
+
+        Scanner scanner = new Scanner(System.in);
+        UserInterface userInterface = new UserInterface(manager, scanner);
+
+        String command;
+        do {
+            userInterface.start();
+            command = scanner.nextLine();
+            switch (command) {
+                case Commands.ADD:
+                    userInterface.add();
+                    break;
+                case Commands.DELETE:
+                    userInterface.del();
+                    break;
+                case Commands.VIEW:
+                    userInterface.view();
+                    break;
+                case Commands.FIND:
+                    userInterface.find();
+                    break;
+                default:
+                    if (!command.equalsIgnoreCase(Commands.EXIT)) System.out.println("Command does not exist.");
+            }
+        } while (!command.equalsIgnoreCase(Commands.EXIT));
+
+        try {
+            FileOutputStream outputStream = new FileOutputStream("src/main/resources/save.ser");
```

Comment:
"src/main/resources/save.ser" this string is used multiple times in the file. create a variable for it ( hardcoded value)